scrape/scrape.go

```diff
@@ -216,8 +288,7 @@ func newScrapePool(cfg *config.ScrapeConfig, app Appendable, jitterSeed uint64,
 		}
 		opts.target.SetMetadataStore(cache)
 
-		return newScrapeLoop(
-			ctx,
+		return newScrapeLoop(ctx,
```

Comment:
```suggestion
		return newScrapeLoop(
		        ctx,
```

Reply:
Yes, that's indeed the Prometheus style.